go/server/cron.go

```diff
@@ -57,6 +60,12 @@ var (
 	queue            executionQueue
 )
 
+func (ei executionIdentifier) equalWithoutUUID(id executionIdentifier) bool {
+	ei.UUID = ""
+	id.UUID = ""
```

Comment:
Would this permanently set the UUIDs to empty string once `equalWithoutUUID` is called? Don't have the context on whether or not that is intentional!

Reply:
Ha! You're totally right. I need to brush off more golang.